plenum/server/replicas.py

```diff
@@ -47,6 +47,8 @@ def add_replica(self, instance_id) -> int:
                                description),
                        extra={"tags": ["node-replica"]})
 
+        self._monitor.reset()
```

Comment:
Please add INFO level log message about resetting the monitor

Reply:
ok